src/components/Body.jsx

```diff
@@ -0,0 +1,39 @@
+import { 
+  Center,
+  Container,
+  Flex, 
+  Heading, 
+} from '@chakra-ui/react';
+import JokesCard from './JokesCard';
+import ParametersForm from './ParametersForm';
+
+export default function Body() {
+  return (
+    <>
+      <Container maxW='container.xl'>
+        <Heading 
+          as='h4' 
+          size='md' 
+          mb={3}
+        >
+          let's generate:
+        </Heading>
+
+        <Flex
+          mt={3} 
+          mb={14} 
+          alignItems='center' 
+          justifyContent='center'
+        >
+          <Center as='form' w={['100%', '75%']} flexDirection={['column', 'row']}>
```

Comment:
I would expect the `form` element to be part of the `ParametersForm` component, not above on a layouting component:
```
          <Center as='form' w={['100%', '75%']} flexDirection={['column', 'row']}>
            <ParametersForm />
          </Center>
```